src/tests/test_dns_recursion.py

```diff
@@ -0,0 +1,12 @@
+from unitest import TestCase
+from src.checks.dns_test_recursion import run
+
+class Test(TestCase):
+    def test_run:
+        # Here we check that RA gives false
+        res = run("google.com", ["ns1.google.com", "ns2.google.com", "ns3.google.com"])
+        assert res["result"]
```

Comment:
This is asserting that it returns true. Conflict with comment. Should it be `assert not res["result"]` ?

Reply:
Okay, I fixed it for both test programs and correct the comment for them